### PR TITLE
EVG-15557 Update version message with patch description

### PR DESCRIPTION
**Pull request by bynn**
[EVG-15557](https://jira.mongodb.org/browse/EVG-15557)

### Description 
update version message so the patch page shows update name 

### Testing 
image

### CODE REVIEW

File: graphql/util.go [hunk 1]
```diff
@@ -235,6 +235,9 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 	if err = p.SetDescription(patchUpdateReq.Description); err != nil {
 		return errors.Wrap(err, "Error setting description"), http.StatusInternalServerError, "", ""
 	}
+	if err = model.UpdateVersionMessage(p.Version, patchUpdateReq.Description); err != nil {
```
Comment (ablack12) [inline graphql/util.go:238]: We should only do this if p.Version != "" (so should move to 266)

**Review by ablack12 — CHANGES_REQUESTED** [review]

File: model/version_db.go [hunk 2]
```diff
@@ -314,6 +314,17 @@ func VersionUpdateOne(query interface{}, update interface{}) error {
 	)
 }
 
+func UpdateVersionMessage(versionId, message string) error {
```
Comment (malikchaya2) [inline model/version_db.go:317]: in the comment on the ticket, it says that the problem is  `that the version gets the update but the patch does not.`. Is it really the opposite, that the patch gets it but not the version?

Comment (malikchaya2) [reply model/version_db.go:317]: that would make sense since it shows up on "my patches" but not the actual patch page.

**Review by malikchaya2 — APPROVED** [review]
lgtm after Annie's comments are addressed.

File: graphql/util.go (hunk 1 above)
Comment (bynn) [reply graphql/util.go:238]: done

File: model/version_db.go (hunk 2 above)
Comment (bynn) [reply model/version_db.go:317]: yeah, the patch in the db is being updated but the UI uses the version page displays the version message which isnt being updated

File: graphql/util.go [hunk 3]
```diff
@@ -259,6 +259,17 @@ func SchedulePatch(ctx context.Context, patchId string, version *model.Version,
 		if version == nil {
 			return errors.Errorf("Couldn't find patch for id %v", p.Version), http.StatusInternalServerError, "", ""
 		}
+
+		v, err := model.VersionFindOneId(p.Version)
```
Comment (ablack12) [inline graphql/util.go:263]: We don't need to call this here; you can see at line 259 that we've already checked that the  version isn't nil (it's passed into schedule patch)

File: rest/route/patch_test.go [hunk 4]
```diff
@@ -691,6 +691,10 @@ buildvariants:
 		PatchedParserProject: config,
 	}
 	require.NoError(t, unfinalized.Insert())
+	// version := serviceModel.Version{
```
Comment (ablack12) [inline rest/route/patch_test.go:694]: Should remove this

**Review by ablack12 — CHANGES_REQUESTED** [review]

File: graphql/util.go (hunk 3 above)
Comment (bynn) [reply graphql/util.go]: done

File: rest/route/patch_test.go (hunk 4 above)
Comment (bynn) [reply rest/route/patch_test.go]: sorry, done

**Pull request merged**